docs/_content/productvisie/index.md

```diff
@@ -0,0 +1,285 @@
+---
+title: "Productvisie ZGW API's"
```

Comment:
Dit is een duplicate van `docs/_content/productivisie.md`, toch? Welke kan weg?

Reply:
Ik heb de content van productvisie.md overgezet naar docs/_content/productvisie/index.md, want: jekyll....
Ben productvisie.md vergeten te verwijderen, ga ik doen

Reply:
Als het goed is nu opgelost